libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -22,6 +21,7 @@
 from libcodechecker.analyze.analyzers import result_handler_clang_tidy
 from libcodechecker.analyze.analyzers import result_handler_plist_to_db
 from libcodechecker.analyze.analyzers import result_handler_plist_to_stdout
+from libcodechecker.cmd.cmd_line_client import setup_client
```

Comment:
Yes I know this is not the way it should be handled but I did not wanted to do more refactoring during the development and importing this way was easy to get the functionality.

I've factored out to an `api_client` package which can be used by the command line client and the store.